### ISSUE TITLE
ipfsHash not returned by Identity query

### ISSUE THREAD

**Issue by mikeshultz** [OriginProtocol/origin#3695]
Identity query doesn't return `ipfsHash` when it's included in the requested params of the Identity/SkinnyIdentity queries.

**Comment by franckc** [comment]
Yes I noticed that too when testing yesterday. I'll take a stab at it since I'm anyway looking at this identity stuff...
